### ISSUE TITLE
feat:  permissions only as numbers

### ISSUE THREAD

**Issue by tcurdt** [cafkafk/eza#889]
while the usual display of permissions as `.rw-r-----` is nice and dandy,
for some it might be shorter and clearer to see the actual number.

`.rw-r-----` => `0640`

That would be nice feature to have a concise and easy to read display.
And there already is 

```
eza -la --octal-permissions
0640 .rw-r----- 128 root 10 Mar 15:30 foo.env
```

But why does it print both? I only want to see the octal permissions.

**Comment by Delapouite** [comment]
I agree this is not intuitive at the moment, but you can achieve what you want with:
```
eza -la --octal-permissions --no-permissions
```

That said, the `--no-permissions` flag may need to be renamed to something less ambiguous.

**Comment by ShadiZade** [comment]
I would add that the octal permission numbers should be in different colors, not one uniform color as it is now. For example, in `0644`, the 0 could be white, the 6 purple, and the 4s yellow. This would make it much easier to read (and just plain nicer), and would accord with the colors of the full `.rw-r-----` permission format.
